internal/provider/resource_nd_multi_cluster_connectivity.go

```diff
@@ -0,0 +1,652 @@
+		deletePayload["force"] = true
+		deletePayload["credentials"] = map[string]string{
+			"loginDomain": data.ClusterLoginDomain.ValueString(),
+			"password":    data.ClusterPassword.ValueString(),
+			"user":        data.ClusterUsername.ValueString(),
+		}
+
+	}
+	payload, _ := json.Marshal(deletePayload)
+	jsonPayload, _ := gabs.ParseJSON(payload)
+	r.client.DoRestRequest(ctx, &resp.Diagnostics, fmt.Sprintf("%s/%s/remove", clusterPath, data.Id.ValueString()), "POST", jsonPayload)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	// Detaching the cluster from the controller may take a few seconds.
+	time.Sleep(5 * time.Second)
+	tflog.Debug(ctx, fmt.Sprintf("End delete of resource nd_multi_cluster_connectivity with id '%s'", data.Id.ValueString()))
+}
+
+func (r *ClusterResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	tflog.Debug(ctx, "Start import state of resource: nd_multi_cluster_connectivity")
+	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
+
+	var stateData *ClusterResourceModel
+	resp.Diagnostics.Append(resp.State.Get(ctx, &stateData)...)
+
+	tflog.Debug(ctx, fmt.Sprintf("Import state of resource nd_multi_cluster_connectivity with id '%s'", stateData.Id.ValueString()))
+	tflog.Debug(ctx, "End import of state resource: nd_multi_cluster_connectivity")
+}
+
+func getClusterCreateJsonPayload(ctx context.Context, diags *diag.Diagnostics, data *ClusterResourceModel, method string) *gabs.Container {
+	payloadMap := map[string]interface{}{}
+
+	clusterType := ""
+	if !data.ClusterType.IsNull() && !data.ClusterType.IsUnknown() {
+		clusterType = data.ClusterType.ValueString()
+	}
+
+	fabricName := ""
+	if !data.FabricName.IsNull() && !data.FabricName.IsUnknown() {
+		fabricName = data.FabricName.ValueString()
+	}
+
+	if clusterType == "apic" {
+		clusterType = "APIC"
+		aciMap := map[string]interface{}{}
+		if !data.LicenseTier.IsNull() && !data.LicenseTier.IsUnknown() {
+			aciMap["licenseTier"] = data.LicenseTier.ValueString()
+		}
+		aciMap["name"] = fabricName
+
+		if !data.SecurityDomain.IsNull() && !data.SecurityDomain.IsUnknown() {
+			aciMap["securityDomain"] = data.SecurityDomain.ValueString()
+		}
+		if !data.ValidatePeerCertificate.IsNull() && !data.ValidatePeerCertificate.IsUnknown() {
+			aciMap["verifyCA"] = data.ValidatePeerCertificate.ValueBool()
+		}
+
+		orchestrationStatus := "disabled"
+		telemetryStatus := "disabled"
+		if !data.Features.IsNull() && !data.Features.IsUnknown() {
+			featureStrings := make([]string, 0)
+			data.Features.ElementsAs(ctx, &featureStrings, false)
+			for _, feature := range featureStrings {
+				switch feature {
+				case "telemetry":
+					telemetryStatus = "enabled"
+				case "orchestration":
+					orchestrationStatus = "enabled"
+				}
+			}
+		}
+
+		aciMap["orchestration"] = map[string]interface{}{
+			"status": orchestrationStatus,
+		}
+
+		epgDn := ""
+		telemetryNetworkType := "outband"
```

Comment:
so when features are not provided config we default to "" and "outband"? is API not handling this default?